<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Levelscreen</class>
 <widget class="QMainWindow" name="Levelscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>90</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose difficulty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose group</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mediumButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hardButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="extremeButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Extreme</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="easyButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="veryhardButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Very hard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
